integration_test/pg/test_helper.exs

```diff
@@ -1,6 +1,10 @@
 Logger.configure(level: :info)
 ExUnit.start
 
+# Configure Ecto for support and tests
```

Comment:
@mobileoverlord you will have to set those too and you can remove the lock_for_update down below.